bin/alluxio-start.sh

```diff
@@ -52,6 +52,11 @@ ensure_dirs() {
     echo "ALLUXIO_LOGS_DIR: ${ALLUXIO_LOGS_DIR}"
     mkdir -p ${ALLUXIO_LOGS_DIR}
   fi
+  if [[ ! -d "${ALLUXIO_LOG_SERVER_BASE_LOGS_DIR}" ]]; then
+    echo "ALLUXIO_LOG_SERVER_BASE_LOGS_DIR: ${ALLUXIO_LOG_SERVER_BASE_LOGS_DIR}"
+    mkdir -p ${ALLUXIO_LOG_SERVER_BASE_LOGS_DIR}/master_logs
+    mkdir -p ${ALLUXIO_LOG_SERVER_BASE_LOGS_DIR}/worker_logs
```

Comment:
only need to make the base directory now

Reply:
We should only create this directory if someone launches the log server

Reply:
Done.